data/json/vitrification_effect_on_condition.json

```diff
@@ -0,0 +1,397 @@
+                },
+                "EOC_queue_vitrified_int_drain"
+              ]
+            }
+          },
+          {
+            "case": 4,
+            "effect": {
+              "run_eocs": [
+                {
+                  "id": "EOC_vitrified_ambient_message_4",
+                  "effect": [
+                    { "u_message": "Vitrifaction_Ambient_4", "snippet": true, "type": "good" },
+                    { "math": [ "u_vitri_glassed", "++" ] },
+                    { "u_add_morale": "morale_vitri_placid", "bonus": 10, "max_bonus": 99 }
+                  ]
+                },
+                "EOC_queue_vitrified_int_drain"
+              ]
+            }
+          },
+          {
+            "case": 5,
+            "effect": {
+              "run_eocs": [
+                {
+                  "id": "EOC_vitrified_ambient_message_5",
+                  "effect": [
+                    { "u_message": "Vitrifaction_Ambient_5", "snippet": true, "type": "good" },
+                    { "math": [ "u_vitri_glassed", "++" ] },
+                    { "u_add_morale": "morale_vitri_placid", "bonus": 15, "max_bonus": 99 }
+                  ]
+                },
+                "EOC_queue_vitrified_int_drain"
+              ]
+            }
+          },
+          {
+            "case": 6,
+            "effect": {
+              "run_eocs": [
+                {
+                  "id": "EOC_vitrified_ambient_message_6",
+                  "effect": [
+                    { "u_message": "Vitrifaction_Ambient_6", "snippet": true, "type": "good" },
+                    { "math": [ "u_vitri_glassed", "++" ] },
+                    { "u_add_morale": "morale_vitri_placid", "bonus": 20, "max_bonus": 99 }
+                  ]
+                },
+                "EOC_queue_vitrified_int_drain"
+              ]
+            }
+          },
+          {
+            "case": 7,
+            "effect": [ { "u_message": "Vitrifaction_Final", "snippet": true, "popup": true, "type": "good" }, { "u_set_hp": 0 } ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "vitrified_bleed_check",
+    "recurrence": 10,
+    "condition": { "and": [ { "u_has_effect": "VITRIFYING" }, { "u_has_effect": "bleed" } ] },
+    "effect": [
+      {
+        "switch": { "math": [ "u_vitri_vitrified" ] },
+        "cases": [
+          { "case": 0, "effect": { "u_message": "Vitrifaction_Bleed_1", "snippet": true, "type": "good" } },
+          { "case": 3, "effect": { "u_message": "Vitrifaction_Bleed_2", "snippet": true, "type": "good" } },
+          { "case": 5, "effect": { "u_message": "Vitrifaction_Bleed_3", "snippet": true, "type": "good" } }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "vitrified_death",
+    "eoc_type": "AVATAR_DEATH",
+    "condition": { "u_has_effect": "VITRIFYING" },
+    "effect": { "u_message": "Vitrifaction_Final", "snippet": true, "type": "good" }
```

Comment:
Do you mean for this to be type: good?

Reply:
yeah, this one's intentional. if the player reaches the point where either the intensity of the effect reaches seven, or they otherwise manage to hurt the mental construct of themselves badly enough to die, the last thing they feel about anything before their mind turns off is that a wonderful thing is happening to them.